deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

```diff
@@ -97,7 +97,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: hostpath
-          image: k8s.gcr.io/sig-storage/hostpathplugin:v1.7.2
+          image: registry.k8s.io/sig-storage/hostpathplugin:v1.7.2
```

Comment:
Can you add line 87 to registry.k8s.io/sig-storage/csi-snapshotter: v5.0.1

Reply:
@xing-yang Done

Reply:
@xing-yang is `registry.k8s.io` preferred over `k8s.gcr.io`? is it documented somewhere ?